distribution/docs/src/main/resources/content/_sources/confluence-federated-source.adoc

```diff
@@ -39,15 +39,15 @@ Configure an Existing Confluence Federated Source through the ${admin-console}:
 . Select the *Sources* tab.
 . Select the name of the source to edit.
 
-See <<Confluence_Federated_Source, Confluence Federated Source configurations>> for all possible configurations.
+See <<{application-prefix}Confluence_Federated_Source, Confluence Federated Source configurations>> for all possible configurations.
 
 [IMPORTANT]
 ====
-If an additional attribute is not part of the Confluence metacard type or <<_attribute_injection_definition,injected>>, the attribute will not be added to the metacard.
+If an additional attribute is not part of the Confluence metacard type or <<{developing-prefix}attribute_injection_definition,injected>>, the attribute will not be added to the metacard.
 ====
 
 .Usage Limitations of the Confluence Federated Source
 Most of the fields that can be queried on Confluence have some sort of restriction on them. Most of the fields do not support the `like` aka `~` operation so the source will convert `like` queries to `equal` queries for attributes that don't support `like`. If the source receives a query with attributes it doesn't understand, it will just ignore them. If the query doesn't contain any attributes that map to Confluence search attributes, an empty result set will be returned.
 
-Depending on your version of Confluence, when downloading attachments you might get redirected to a different download URL. If this happens the default URLResourceReader configuration does not allow redirects and will fail the download. This can be fixed by enabling redirects in the `URLResourceReader` <<_configuring_the_url_resource_reader,configuration>>.
+Depending on your version of Confluence, when downloading attachments you might get redirected to a different download URL. If this happens the default URLResourceReader configuration does not allow redirects and will fail the download. This can be fixed by enabling redirects in the `URLResourceReader` <<{developing-prefix}configuring_the{developing-prefix}url_resource_reader,configuration>>.
```

Comment:
❓ Should this have `{developing-prefix}` in the middle of the link?

Reply:
nope